uktrade/data-hub-frontend
# Add Data Hub feed to personalised dashboard

## Description of change

This adds a Data Hub feed to personalised dashboard that displays the latest update from Help Centre.

## Test instructions

As a user with enabled personalised dashboard, you should navigate to the home page and scroll to the bottom. The "What's new" section should be visible containing a link to an update.

## Screenshots
### Before

![bottom-no-feed](https://user-images.githubusercontent.com/5889630/168755996-e40af7fe-7600-4f53-871b-5f2e6ce771af.png)

### After

![bottom-feed](https://user-images.githubusercontent.com/5889630/168756030-d9a8d4fc-3bd7-4f1b-bcbc-b32b2e467368.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
